composer.json

```diff
@@ -9,12 +9,12 @@
         "name": "Jonathan Shaw",
         "email": "jonathanjfshaw@gmail.com"
     }
-  ],  
+  ],
   "require": {
       "php": "^7.0 || ^8.0",
-      "phpunit/phpunit": "^6.0 || ^7.0 || ^8.0 || ^9.0",
-      "behat/behat": "^3.0.0",
-      "symfony/dependency-injection": "^4.0 || ^5.0 || ^6.0"
+      "phpunit/phpunit": "^6.0 || ^7.0 || ^8.0 || ^9.0 || ^10.0 || ^11.0",
+      "behat/behat": "^3.0.0 || ^3.14.0",

```

Comment:
Isn't this a reundant requirement? i.e. it should either be ^3.0.0 or ^3.14.0 if versions 3.0-3.13 are no longer valid. ^3.0.0 implies 3.14, 3.15 etc?